util.go

```diff
@@ -61,3 +61,35 @@ func GetBrowserNames() ([]string, error) {
 	sort.Strings(browserNames)
 	return browserNames, nil
 }
+
+// IsBrowserName determines whether the given name string is a valid browser name.
+// Used for validating user-input params for browsers.
+func IsBrowserName(name string) bool {
+	if name == "" {
+		return false
+	}
+	var browsers []string
+	var err error
+	if browsers, err = GetBrowserNames(); err != nil {
+		return false
+	}
+	for _, browser := range browsers {
+		if browser == name {
+			return true
+		}
+	}
+	return false
+}
+
+func abs(x int) int {
```

Comment:
Don't know go, but if https://golang.org/pkg/math/ is part of the standard library, is it most idiomatic to use that or roll your own?

Reply:
math lib works with floats, not ints, arguing that ints are 'trivial to roll your own'.

(I don't like Go in that regard.)